scripts/qa_crowbarsetup.sh

```diff
@@ -3310,11 +3310,7 @@ function oncontroller_run_tempest
         # output the slowest tests from the latest run
         testr slowest
     fi
-    if iscloudver 7plus; then
-        ostestr last --subunit --no-pretty > tempest.subunit.log
-    else
-        testr last --subunit | subunit-2to1 > tempest.subunit.log
-    fi
+    testr last --subunit | subunit-2to1 > tempest.subunit.log
```

Comment:
isn't that supposed to be stestr?

Reply:
Thank you for your comment. oh, good question.

I was thinking to switch to use stestr at the beginning. But I'm actually not sure we can use stestr in SOC7 or the other version at this point. So, I just take a conservative option.

But if we can use stestr, I'm really happy with it. And, I think stestr should work for this usage instead of testr. So, I can try it.
To do so, how can we install stestr package? I can't find the code to install yet..